ExternalRevisions/SPIRV-Cross_repo_revision

```diff
@@ -1 +1 @@
-ccff428086b625241de9f225dab0a53269b4d12c
+5b18a641c7115acea334dd554ef22dc26602b8aa
```

Comment:
After #2494, this is now obsolete, as a newer version is now retrieved. You can just remove this change.